module-services/service-cellular/src/volte/ImsiParserAT.hpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
```

Comment:
Maybe it would be more explicit if you wrote a comment in this file about which country does the file contain? in some cases, it's easy to guess, but not in all cases

Reply:
sounds good to me

Reply:
Done :white_check_mark: